server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/OnlineSchemaChangeFlowableTask.java

```diff
@@ -267,9 +269,9 @@ private ScheduleEntity createScheduleEntity(ConnectionConfig connectionConfig,
         scheduleEntity.setAllowConcurrent(false);
         scheduleEntity.setCreatorId(creatorId);
         scheduleEntity.setOrganizationId(organizationId);
-        // todo project id database id
+        // todo project id
         scheduleEntity.setProjectId(1L);
```

Comment:
what is projectId = 1 means ?

Reply:
project id is not used for osc , but i also query by `databaseService.getBasicSkipPermissionCheck` and and set in scheduleEntity